state/postgres_statement.go

```diff
@@ -104,3 +104,11 @@ func (stmt DiffedPostgresStatementStats) Add(other DiffedPostgresStatementStats)
 		BlkWriteTime:      stmt.BlkWriteTime + other.BlkWriteTime,
 	}
 }
+
+type QueryIdentity struct {
+	QueryID     int64
```

Comment:
I don't think this field is necessary - are we using it anywhere? (its already in the key of the map)

Reply:
(marking as resolved since there's the same comment elsewhere)